docs/getting-started/wallets.md

```diff
@@ -28,21 +33,24 @@ There are many different applications that can interact with your public and/or
 
 ### Permissionless wallet apps
 
-You can visit [bittensor.com/scan](https://bittensor.com/scan) and enter a coldkey public key view public information about any wallet. The browser hence is able to act as a kind of permissionless wallet application to display public information about wallets.
+You can visit [bittensor.com/scan](https://bittensor.com/scan) and enter a coldkey public key to view public information about any wallet.
+The browser in this case is able to act as a kind of permissionless wallet application to display public information about wallets.
 
 ### Staking apps
 
-Several applications exist that can interact securely with your coldkey, meaning essentially that you can load your coldkey into the application without having to have the key stored in unencrypted form on your disk. In theory this means that without your encryption password, it is impossible to steal your key even if someone accesses your device.
+Several applications exist that can interact securely with your coldkey. This basically means that you can load your coldkey into the application without having to have the key stored in an unencrypted form on your disk.
+In theory this means that without your encryption password, it is impossible to steal your key—even if someone accesses your device.
 
-However, it must be considered that using your coldkey on a device offers other ways for attackers to steal your key, even without decrypting it. See [Coldkey workstation security](./coldkey-hotkey-security#coldkey-workstation)
+However, still consider that using your coldkey on a device offers other ways for attackers to steal your key, even without decrypting it.
+See [Coldkey workstation security](./coldkey-hotkey-security#coldkey-workstation).
 
-This includes:
+Application vectors that can expose your coldkey include:
```

Comment:
Nah, the idea is that these are the secure apps. The insecure vectors would be other things on your computer. these are considered intrinsically secure.
```suggestion
Secure wallet apps supported by Opentensor Foundation include:
```

Reply:
ah got it, the way it went from talking about other ways for attackers to steal your key to "This includes" made me think the opposite.